src/Aspire.Dashboard/Extensions/StringExtensions.cs

```diff
@@ -49,4 +31,29 @@ static bool IsValidHtmlIdCharacter(char c)
             return char.IsLetterOrDigit(c) || c == '_' || c == '-';
         }
     }
+
+    /// <summary>
+    /// Returns the two initial letters of the first and last words in the specified <paramref name="name"/>.
+    /// If only one word is present, a single initial is returned. If <paramref name="name"/> is null, empty or
+    /// white space only, <paramref name="defaultValue"/> is returned.
+    /// </summary>
+    public static string? GetInitials(this string name, string? defaultValue = default)
```

Comment:
I think this can be annotated with:

```suggestion
    [NotNullIfNotNull(nameof(defaultValue))]
    public static string? GetInitials(this string name, string? defaultValue = default)
```

Reply:
Had to specify `[return: NotNullIfNotNull(nameof(defaultValue))]`, but you're right that's good.